### PR TITLE
WIP (do not merge yet): rework model-in-image sample

### PR DESCRIPTION
**Pull request by tbarnes-us**
Untested rework of the model-in-image sample.

- Update readme & scripts.
- Rename files.

Testing is in progress...

### CODE REVIEW

File: kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build.sh [hunk 1]
```diff
@@ -22,97 +21,61 @@ export SERVER_JRE=V982783-01.zip
...
+if [ ! -f "${FMW_INSTALLER}" ] \
+   && [ "${DOMAINTYPE}" == "RestrictedJRF" -o "${DOMAINTYPE}" == "JRF" ] ; then
+  echo "Directory ${WORKDIR} does not contain ${FMW_INSTALLER}." && exit 
 fi
 
-#
-#
-cp -R * ${WORKDIR}
-cd ${WORKDIR}
-#
```
Comment (tbarnes-us) [inline kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build.sh]: I've moved the above into readme.md instead of having the script do it.

**Comment by TheFrogPad** [comment]
Hi Tom, Johnny, is there a reason the sample must have you download and create the images from the base JDK and WebLoigic/FMW intallers?

I believe all the other samples start with the images in OCR and then add in what is needed?

**Comment by jshum2479** [comment]
> Hi Tom, Johnny, is there a reason the sample must have you download and create the images from the base JDK and WebLoigic/FMW intallers?
> 
> I believe all the other samples start with the images in OCR and then add in what is needed?

This is an example I want to show the imagetool can be used as an alternative.  The imagetool allows you to start from scratch.

**Comment by tbarnes-us** [comment]
> > Hi Tom, Johnny, is there a reason the sample must have you download and create the images from the base JDK and WebLoigic/FMW intallers?
> > I believe all the other samples start with the images in OCR and then add in what is needed?
> 
> This is an example I want to show the imagetool can be used as an alternative. The imagetool allows you to start from scratch.

There's merit in demonstrating the image tool to start from scratch, but starting from scratch is a very lengthy process, so I think it'd be helpful to also provide a way to start directly from the image.   Let's think about doing that for the final version, and maybe moving the original image creation step to a common location that could be shared by the other samples as well as this one...

**Pull request merged**